### ISSUE TITLE
supported_cuba and contained_cuba methods for CUDS datasets 

### ISSUE THREAD

**Issue by nathanfranklin** [simphony/simphony-common#217]
from @itziakos :  https://publicwiki-01.fraunhofer.de/SimPhoNy-Project/index.php/Supported_CUBA_keys_for_container

**Comment by kitchoi** [comment]
I think this is related:
In [abc_check_mesh](https://github.com/simphony/simphony-common/blob/36e8a752250b0495f5ee03ff6f42d48368b1bcf9/simphony/testing/abc_check_mesh.py#L414), should `self.supported_cuba()` be supplied too? (i.e. `create_points(restrict=self.supported_cuba()`)

**Comment by nathanfranklin** [comment]
@kitchoi ,  I totally agree.  the testing frameworks (`abc_check_mesh`, `abc_check_lattice`..) should be updated to make use of the new `supported_cuba(item_type)` methods

**Comment by ahashibon** [comment]
Since the SSB is finalising the CUDS design, this should be put on hold until we define the CUDS more accurately, please remove from the current milestone.
